servant-util/examples/Books.hs

```diff
@@ -1,32 +1,52 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DerivingStrategies #-}
+
 module Books where
 
 import Universum
 
 import Data.Aeson (FromJSON, ToJSON)
 import Data.Aeson.TH (defaultOptions, deriveJSON)
+import Data.Swagger
 import Fmt (Buildable (..), (+|), (|+))
 import qualified Network.Wai.Handler.Warp as Warp
 import Servant (FromHttpApiData, Get, JSON, PostCreated, QueryParam, ReqBody, Server, serve,
                 (:<|>) (..), (:>))
+import Servant.Swagger
+import Servant.Swagger.UI
```

Comment:
Our style guide says to use explicit import lists for third-party libraries, and I think this would be even more valuable for an example.